### ISSUE TITLE
Source maps broken on default dotnet core ASP.NET (Angular) app 

### ISSUE THREAD

**Issue by EricCornelson** [microsoft/vscode-js-debug#309]
Repro steps (VS code insiders latest nightly 2/6):
1. Create a new ASP.NET Angular project using `dotnet new angular`
2. Build and run the project (uses https by default)
3. Set a breakpoint in app.component.ts
4. Create a Chrome launch config using v3 and start debugging
5. Breakpoints in .ts files not bound, the following error message appears in logs:

```js
{
	"tag": "sourcemap.parsing",
	"timestamp": 1581013628051,
	"message": "Error fetching sourcemap",
	"metadata": {
		"message": "unable to verify the first certificate",
		"stack": "Error: unable to verify the first certificate
			TLSSocket.onConnectSecure (_tls_wrap.js:1317:34)
			TLSSocket.emit (events.js:200:13)
			TLSSocket._finishInit (_tls_wrap.js:792:8)
			TLSWrap.ssl.onhandshakedone (_tls_wrap.js:606:12)",
		"code": "UNABLE_TO_VERIFY_LEAF_SIGNATURE"
	},
	"level": 2
}
```

[vscode-debugadapter.log](https://github.com/microsoft/vscode-js-debug/files/4167242/vscode-debugadapter.log)

**Comment by EricCornelson** [comment]
Confirmed the same thing also happens to default react template as well. Workaround is to disable https redirect and use http only

**Comment by llgcode** [comment]
I've reproduced the issue on my environment :

`Could not read source map for https://********: unable to verify the first certificate`

We debug our web site that has a URL that uses certificate issued by an enterprise root Certificate.

I've set "NODE_EXTRA_CA_CERTS" environment variable to the path of the root certificate. I've tried with a nodejs client sample to verified that file is ok. But with the new js debugger it's not possible to debug
